test/uk/gov/hmrc/cgtpropertydisposalsfrontend/connectors/upscan/UpscanConnectorImplSpec.scala

```diff
@@ -79,49 +108,46 @@ class UpscanConnectorImplSpec extends AnyWordSpec with Matchers with MockFactory
     implicit val hc: HeaderCarrier = HeaderCarrier()
     val reference                  = sample[UploadReference]
     val upload                     = sample[UpscanUpload]
-    val baseUrl                    = "https://host3:123/cgt-property-disposals"
 
     "Initialising" must {
-      val expectedUrl = "https://host2:123/upscan/v2/initiate"
+      val expectedUrl = "/upscan/v2/initiate"
       val mockSuccess = Call("GET", "/mock-success")
       val mockFailure = Call("GET", "/mock-fail")
 
       val payload = UpscanInitiateRequest(
-        s"$baseUrl/upscan-call-back/upload-reference/${reference.value}",
+        s"https://host3:123/cgt-property-disposals/upscan-call-back/upload-reference/${reference.value}",
```

Comment:
Once both services are wiremock based, replace the hardcoded host/port with wiremock host/port